nav2_bringup/bringup/README.md

```diff
@@ -1,21 +1,21 @@
 # nav2_bringup
 
-The `nav2_bringup` package is an example bringup system for navigation2 applications.
+The `nav2_bringup` package is an example bringup system for Navigation2 applications.
 
-Notes: (June 2019, Dashing Release)
-* We recommend doing this on a Ubuntu 18.04 installation. We have build issues on 16.04. (see https://github.com/ros-planning/navigation2/issues/353)
-* This stack and ROS2 are still in heavy development and there are some bugs and stability issues being worked on, so please do not try this on a robot without taking *heavy* safety precautions. THE ROBOT MAY CRASH!
-* It is recommended to start with simulation using Gazebo before proceeding to run on a physical robot
-
-Install and build our code by following this guide:
-https://github.com/ros-planning/navigation2/blob/master/doc/BUILD.md
+### Pre-requisites:
+* [Install ROS 2](https://index.ros.org/doc/ros2/Installation/Dashing/)
+* [Install Navigation 2](https://github.com/ros-planning/navigation2/blob/master/doc/BUILD.md)
```

Comment:
You're linking to building from source installation, is there not a document for installing via apt? If not, you can just put that here

`sudo apt install ros-[rosdisto]-navigation2`

The generic running example shouldnt involve source code

Reply:
There is no install via apt documentation yet.  I think I can just rename build.md as build-install.md and put 

`sudo apt install ros-[rosdisto]-navigation2 ` somewhere in there

so we can link other docs to that page. 

Do you think it is better to have separate install.md?

Reply:
I think you should just put the apt install instructions here, I dont think it requires its own file

Reply:
I decided use build instead of install since the next steps tell user to source the setup.bash file in their nav2 workspace.

Reply:
You can instead source `/opt/ros/dashing/setup.bash`, I dont think that's a reason to default to source builds.